### PR TITLE
chore(pipelines): PRs, merge main and upload on release

### PR DESCRIPTION
**Pull request by portellaa**

### CODE REVIEW

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    - uses: ydataai/cancel-workflow-action@0.7.0
+      with:
+        ignore_sha: true
+        access_token: ${{ secrets.ACCESS_TOKEN }}
+
+
+  validate:
+    name: Validate
+    runs-on: ubuntu-20.04
+
```
Comment (andre-lx) [inline .github/workflows/pull_request.yml:30]:
```
needs:
- cancel_previous
```

File: .github/workflows/merge_main.yml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+      env:
+        GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+      with:
+        tag_name: ${{ needs.prepare.outputs.version }}
+        release_name: ${{ needs.prepare.outputs.version }}
+        body: ${{ github.event.head_commit.message }}
+        prerelease: true
+
+
+  draft-release:
```
Comment (andre-lx) [inline .github/workflows/merge_main.yml:65]: please merge the prerelease and draft release to make it faster

**Review by andre-lx — APPROVED** [review]

File: .github/workflows/pull_request.yml (hunk 1 above)
Comment (portellaa) [reply .github/workflows/pull_request.yml:30]: why? the idea is to cancel because it's not necessary to run it fully, but it can start to run right away

Comment (portellaa) [reply .github/workflows/pull_request.yml:30]: also, in this case it doesn't really need to cancel, this will be really quick

**Pull request merged**